src/platform/apollolake/include/platform/interrupt.h

```diff
@@ -84,7 +84,7 @@
 /* IRQ Level 5 bits */
 #define IRQ_BIT_LVL5_LP_GP_DMA1(x)	(24 + x)
 #define IRQ_BIT_LVL5_LP_GP_DMA0(x)	(16 + x)
-#define IRQ_BIT_LVL5_DMIC		6
+#define IRQ_BIT_LVL5_DMIC(x)		(6 + x)
```

Comment:
here, I discuss with keyon. In spec, we only find interrupt pin 6 is for DMIC. pin 7 is for slimbus.
we are confusing about it.

Reply:
@RanderWang APL and CNL have different IRQ mappings at level 5.

Reply:
Yep, I have information for APL and CNL. I'll find out also ICL.

Reply:
I'll fix also Sue. ICL is still open, if no info I'll keep it as CNL.